.github/workflows/api.yml

```diff
@@ -30,4 +30,31 @@ jobs:
       - name: Test massa API
         uses: matt-ball/newman-action@master
         with:
-          collection: api/test/massa_endpoints.postman_collection.json
\ No newline at end of file
+          collection: api/test/massa_endpoints.postman_collection.json
+
+  test-api-robot:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project directory
+        uses: actions/checkout@v3
+      - name: Install RobotFramework tests dependencies
+        working-directory: api/test
+        run: |
+          pip install -r requirements.txt
+      - name: Install dependencies
+        uses: ./.github/actions/install
+        with:
+          os: ubuntu-latest
+      - name: Build thyra binary
+        run: GOOS=linux GOARCH=amd64 CGO_ENABLED=1 CC=gcc CXX=g++ go build cmd/thyra-server/main.go
+      - name: Prepare thyra to be runned
+        run: mkdir /home/runner/.config/thyra && sudo setcap CAP_NET_BIND_SERVICE=+eip ./main && sudo apt install xvfb
```

Comment:
can you move the `apt install xvfb` in the  `Install RobotFramework tests dependencies` step ?